packages/core/forms/src/composables/useAbstractFields.ts

```diff
@@ -0,0 +1,250 @@
+import { computed, ref } from 'vue'
+import {
+  get as objGet,
+  forEach,
+  isFunction,
+  isString,
+  isArray,
+  debounce,
+  uniqueId,
+  uniq as arrayUniq,
+  type DebouncedFunc,
+} from 'lodash'
```

Comment:
suggestion: just add the individual `lodash.*` packages we need instead of the entire dep

Reply:
Curious: Would we consider utilizing `loadash-es` in the future?

Reply:
@adamdehaven do you have an opinion on this?

Reply:
It looks fine if you want to switch

Reply:
@adamdehaven @sumimakito 

Why are we wanting to switch to `-es` out of curiosity? I originally thought the goal of this comment (and switching to `lodash-es`) was to improve bundle size. The [reading](https://lodash.com/per-method-packages) about the per method packages, it sounds like usage isn't recommended and they'll be removed in v5. Checking about recommended import strategies, [research](https://www.blazemeter.com/blog/import-lodash-libraries#lodash-and-lodash-es-import-comparison-table) seems to show `lodash-es` will increase package size. 

The recommended import strategy for `lodash` with only a few functions is one-by-one imports (ex. `import get from 'lodash/get'`), so I've gone ahead and switched to using the `lodash-es` package with one-by-one import strategy and it seems like everything is working as expected.

Is there another reason for using `lodash-es`?

Reply:
> Checking about recommended import strategies, [research](https://www.blazemeter.com/blog/import-lodash-libraries#lodash-and-lodash-es-import-comparison-table) seems to show lodash-es will increase package size.

@kaiarrowood That was an interesting finding. I tried to build `lodash` + one-by-one imports / `lodash-es` + `import { ... } from 'lodash-es'` / `lodash-es` + one-by-one imports with Vite and use `rollup-plugin-visualizer` to inspect the bundle size, it turned out that the render sizes of these `lodash-es` builds were still several bytes smaller than the `lodash` one.

I also did a deeper look at the build artifacts of the three versions and suspected the several bytes difference would be due to UMD overheads like `Object.defineProperty(exports, "__esModule", ...)` containing in the `lodash` build. 🤔